flang/lib/Lower/Intrinsics.cpp

```diff
@@ -777,12 +822,12 @@ mlir::Value IntrinsicLibrary::genMod(mlir::Type resultType,
   if (resultType.isa<mlir::IntegerType>())
     return builder.createHere<mlir::SignedRemIOp>(args[0], args[1]);
 
-  // Use runtime. Note that mlir::RemFOp alos implement floating point
+  // Use runtime. Note that mlir::RemFOp also implement floating point
```

Comment:
Does this mean `RemFOp` is also a floating-point type operation?

Reply:
The "also" is a bit misleading, I just removed it.
Just above, `mlir::SignedRemIOp` is used for integers, the point of the comment here is to justify why we are not using `mlir::RemFOp` to implement the floating point version, given its a perfect match. Its also a note for the future, in case we want to start using mlir intrinsic operation when we have `mlir::FloatType` in the hand (which is probably the majority). For now, I'd rather have a single path.

Reply:
Let's add a TODO in the comment if there is something we want to revisit later. It'll make it easier to find again.